rlkit/samplers/vec_sampler.py

```diff
@@ -82,7 +80,7 @@ def rollout(
                 env_infos=env_info,
             )
 
-        observations = next_observations
+        observations = next_observations[terminals == False]
```

Comment:
For environments with variable episode length, some workers may terminate early than others.